### PR TITLE
New post update function, search is now using the "global" field

### PR DESCRIPTION
**Pull request by annando**
Some time ago we introduced the field "global" in the item table that indicates whether an item has a global copy with "uid = 0". Now there is a functionality that updates old item entries where the "global" field wasn't set.

Additionally the search functionality was changed so that this field is now used.

The existing "post update" function is moved into a new file.

### CODE REVIEW

File: include/cron.php [hunk 1]
```diff
@@ -106,8 +107,8 @@ function cron_run(&$argv, &$argc){
 	// Check every conversation
 	ostatus::check_conversations(false);
 
-	// Set the gcontact-id in the item table if missing
-	item_set_gcontact();
+	// Do post update functions
```
Comment (rabuzarus) [inline include/cron.php]: little better description would be nice. `Do post update functions` explains nothing

Comment (annando) [inline include/cron.php]: Question is: What can I say more? This is simply a call to a function that will call pending post update functions.

Comment (rabuzarus) [inline include/cron.php]: Mhh difficult. I thought you could say more. I don't know exactly what happens. I interpreted it as something to have some post processing for posts which doesn't have the correct format in the db (im weitesten Sinne) - like the former function item_set_gcontact

**Comment by rabuzarus** [comment]
:+1:

**Pull request merged**